tools/generate_mockserver_config.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import glob
+
+
+def get_rule(path: str, body: str, queryStringParameters=None) -> dict:
+    rule = {
+        "httpRequest": {
+            "path": path,
+        },
+        "httpResponse": {
+            "headers": {
+                "Content-Type": ["application/json"]
+            },
+            "body": body
+        }
+    }
+    if queryStringParameters is not None:
+        rule["httpRequest"]["queryStringParameters"] = queryStringParameters
+    return rule
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
usage of ``pathlib`` is highly recommended
you can also do ``path.open("r")`` and ``path.glob("clusters-*")`` as shortcuts